tests/python_package_test/test_basic.py

```diff
@@ -13,6 +14,14 @@
 from .utils import load_breast_cancer
 
 
+def rm_files(files):
+    if not isinstance(files, list):
+        files = [files]
+    for file in files:
+        if os.path.exists(file):
+            os.remove(file)
```

Comment:
As mentioned before, this should be removed.

Reply:
removed via https://github.com/microsoft/LightGBM/pull/4089/commits/2d32b873db3ab7b436e2941389eceb9817290d8d